diff --git a/node_modules/html-to-image/es/embed-webfonts.js b/node_modules/html-to-image/es/embed-webfonts.js
index f6b0d96..133ffb1 100644
--- a/node_modules/html-to-image/es/embed-webfonts.js
+++ b/node_modules/html-to-image/es/embed-webfonts.js
@@ -158,8 +158,8 @@ function normalizeFontFamily(font) {
 function getUsedFonts(node) {
     const fonts = new Set();
     function traverse(node) {
-        const fontFamily = node.style.fontFamily || getComputedStyle(node).fontFamily;
-        fontFamily.split(',').forEach((font) => {
+        const fontFamily = node.style.fontFamily || getComputedStyle(node).fontFamily || '';
+        (fontFamily || '').split(',').forEach((font) => {
             fonts.add(normalizeFontFamily(font));
         });
         Array.from(node.children).forEach((child) => {
@@ -175,7 +175,7 @@ export async function getWebFontCSS(node, options) {
     const rules = await parseWebFontRules(node, options);
     const usedFonts = getUsedFonts(node);
     const cssTexts = await Promise.all(rules
-        .filter((rule) => usedFonts.has(normalizeFontFamily(rule.style.fontFamily)))
+        .filter((rule) => usedFonts.has(normalizeFontFamily(rule.style.fontFamily || '')))
         .map((rule) => {
         const baseUrl = rule.parentStyleSheet
             ? rule.parentStyleSheet.href
